nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -425,12 +504,17 @@ Nav2Panel::Nav2Panel(QWidget * parent)
   // the state of the application will also update. This means that if in the processing
   // states and then goes inactive, move back to the idle state. Vise versa as well.
   ROSActionQTransition * idleTransition = new ROSActionQTransition(QActionState::INACTIVE);
-  idleTransition->setTargetState(running_);
+  idleTransition->setTargetState(running_nav_through_poses_);
   idle_->addTransition(idleTransition);
+  //! @todo add a transition for follow WP here?
+
+  ROSActionQTransition * runningNavThroughPosesTransition = new ROSActionQTransition(QActionState::ACTIVE);
+  runningNavThroughPosesTransition->setTargetState(idle_);
+  running_nav_through_poses_->addTransition(runningNavThroughPosesTransition);
```

Comment:
No the idle accumulated_state_transition is taken care somewhere in the below, you do not have to do it here..